### PR TITLE
Add "A new home for the Project Fugu API Showcase" article

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add "A new home for the Project Fugu API Showcase" article.
- Redirect `/blog/fugu-showcase/` to `/fugu-showcase/`.
- Review happened in the associated [Google doc](https://docs.google.com/document/d/1kW3OSn2OaQBlF_hwMeqViAHup557WKqFtFzVcZKLvxQ/edit).

### CODE REVIEW

File: site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - thomassteiner
+tags:
+  - capabilities
+---
+
+The cross-company Capabilities Project (code name [Project Fugu](/capabilities/)) at Google has the objective of making it possible for web apps to do anything platform-specific apps can. Apart from Google, project partners include Microsoft, Intel, Samsung, and others. The project enables amazing web applications like [Photoshop](https://web.dev/ps-on-the-web/) by exposing the capabilities of the underlying operating systems to the web platform, while maintaining user security, privacy, trust, and other core tenets of the web.
+
+But what are examples of apps that make use of these capabilities? Our answer to this question is the [Project Fugu API Showcase](/fugu-showcase/). It is sourced by community submissions, and contains a filterable list of apps that make use of one or more of the Fugu APIs.
+
+{% Aside 'success' %}
```
Comment (rachelandrew) [inline site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md:20]: these two seem a bit odd as "success" as that doesn't really describe their contents. I think they could just be regular asides.

File: site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{% endAside %}
+
+
+After living in an embedded iframe that didn't really allow the showcase to fully shine, we have now worked on migrating it to a new home, and have polished its look and feel a fair bit, too. Find the Project Fugu API Showcase at its new location [developer.chrome.com/fugu-showcase](/fugu-showcase/)!
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/c7is7cp31DfaKl3mPt2q.png", alt="The Project Fugu API Showcase.", width="800", height="749" %}
+
+Launch each app by clicking its name, screenshot, or the **Launch app** link. For many apps, you can also see the source code by clicking **View source**. On browsers that support the [Web Share API](https://developer.mozilla.org/docs/Web/API/Web_Share_API), click **Share app** to do exactly that. As a special inception Easter egg, the [Project Fugu API Showcase is of course contained in the Project Fugu API Showcase](/fugu-showcase/#developer.chrome.com!fugu-showcase).
+
+{% Aside 'success' %}
```
Comment (rachelandrew) [inline site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md:31]: this one as well

**Review by rachelandrew — APPROVED** [review]
This is fine other than the suggestion that success doesn't really describe how you are using those asides.

File: site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md (hunk 1 above)
Comment (tomayac) [inline site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md:20]:
```suggestion
{% Aside %}
```

File: site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md (hunk 2 above)
Comment (tomayac) [inline site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md:31]:
```suggestion
{% Aside %}
```

File: site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md (hunk 1 above)
Comment (tomayac) [reply site/en/blog/a-new-home-for-the-project-fugu-api-showcase/index.md]: Fixed.

**Pull request merged**